config/routes.rb

```diff
@@ -1,4 +1,35 @@
 Rails.application.routes.draw do
+
+  root "welcome#index"
+
+  get 'welcome/index'
```

Comment:
You probably don't need line 5 since line 3 handles the same functionality but for the root path.